main.nf

```diff
@@ -1110,7 +1113,8 @@ process proteomicslfq {
                    -out out.mzTab \\
                    -threads ${task.cpus} \\
                    ${msstats_present} \\
-		   ${triqler_present} \\
+                   ${triqler_present} \\
+                   ${decoys_present} \\
```

Comment:
@jpfeuffer do you think the mzTab is exporting by default also the decoys?

Reply:
Yes, if decoys are activated, the mzTab should have them too.

Reply:
You can check the GitHub Actions artifacts to see how the output files look like.